src/tests/system/tests/test_schema.py

```diff
@@ -42,22 +46,21 @@ def test_schema__ldap_extra_attrs_filled(client: Client, provider: GenericProvid
         pytest.fail(f"Exception shouldn't be raised but we got {type(e)}: str(e)")
 
     result = client.tools.getent.passwd("tuser")
-    assert result is not None
-    assert result.name == "tuser"
+    assert result is not None, "User not found!"
+    assert result.name == "tuser", f"User 'tuser' name is not the expected value `{result.name}`!"
 
 
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
-def test_schema__ldap_extra_attrs_check_ldb(client: Client, provider: GenericProvider):
+def test_schema__user_extra_attributes_are_written_to_ldb_cache(client: Client, provider: GenericProvider):
```

Comment:
I was explained some time ago that system tests are black box tests. With that in mind, this test should not exist.

Reply:
GLADLY